src/communicate/main.py

```diff
@@ -1661,7 +1662,7 @@ def record_search_whiteness(user: int, search_id: int, seach_following_flag) ->
             cur.execute("""DELETE FROM user_pref_search_whitelist WHERE user_id=%(user)s and search_id=%(search_id)s;""", {'user':user, 'search_id':search_id})
         return None
 
-    logging.info('manage_search_whiteness..callback_query='+str(callback_query))
+    logging.info('callback_query='+str(callback_query))
```

Comment:
It's a minor thing, but I propose to use more f-strings: it's more readable and you don't need str()